modules/subscriptions/subscriptions.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { GlobalCore } from "@/core/module/module.types";
+import subscriptiosService from "@/services/app/subscriptios.service";
+import styles from "./styles/subscriptions.module.css";
+
+const Subscriptions = () => {
+  const { t } = useTranslation();
+  const [isLoading, setIsLoading] = useState(false)
+
+  const handleSubscribe = async () => {
+    setIsLoading(true)
+    const { url } = await subscriptiosService.getSubscriptionLink();
+    if (url) {
+      window.location.href = url;
+    }
+    setIsLoading(false)
+  };
+
+  return (
+    <>
+      <main className={styles.mainContent}>
+        <h1 className={styles.pricingTitle}>
+          Find the pricing plan for your business
+        </h1>
+        <p className={styles.pricingSubTitle}>
+          Select the perfect paln for your business to grow
+        </p>
+        <div className={styles.pricingCardsWrapper}>
+          <div className={styles.pricingCard}>
+            <div className={styles.priceTitle}>FREE</div>
+            <div className={styles.priceTag}>
+              <span className={styles.priceCurrency}>€</span>0
+              <span className={styles.priceInterval}>/month</span>
+            </div>
+            <hr className={styles.priceHr} />
+            <ul className={styles.priceFeatList}>
+              <li>Benefit 1</li>
+            </ul>
+            <button onClick={handleSubscribe} disabled={isLoading} className={styles.priceButton}>
+              {t("TRY IT NOW")}
```

Comment:
Check the lang, it had changed, now you want to do `subscriptions.purchase` or something like this (instead of the text as a key)